cpu/native/startup.c

```diff
@@ -390,6 +392,13 @@ extern init_func_t __init_array_start;
 extern init_func_t __init_array_end;
 #endif
 
+static void reset_handler(int signo)
```

Comment:
I changed this to 
```
static void _reset_handler(void)
{
    pm_reboot();
}
```

and added:

```
register_interrupt(SIGUSR1, _reset_handler);
```

below and it works on FreeBSD, the `signal(...)` is not needed